base_location_nuts/models/res_partner_nuts.py

```diff
@@ -24,6 +24,7 @@ class ResPartnerNuts(models.Model):
     # Parent hierarchy
     parent_id = fields.Many2one(comodel_name='res.partner.nuts',
                                 ondelete='restrict')
+    parent_path = fields.Char(index=True)
```

Comment:
If you switch this one to the new system, you have to:

* Remove `parent_left` field.
* Remove `parent_right` field.
* Add migration script for recomputing the field.

See in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0#tasks-to-do-in-the-migration full explanation, and ask me if you want about migration script.